pages/models.py

```diff
@@ -162,6 +163,11 @@ def get_leafs(self):
             ' WHERE pages_relationship.orig_element_id = pages_pageelement.id)'\
             ' = 0'])
 
+    @staticmethod
+    def followed_by(user):
+        page_element_slugs = user.follows.all().values_list('element__slug', flat=True)
+        return PageElement.objects.filter(slug__in=page_element_slugs)
```

Comment:
an we not replace the 2 SQL queries this generate by `PageElement.objects.filter(followers__user=user)`?